spec/systems/sessions_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Sessions', type: :system do
+  describe 'login' do
+    context 'validates screen' do
+      before(:each) do
+        visit login_path
+      end
+
+      it 'login screen should show correct field and button' do
+        within 'form' do
+          expect(page).to have_field('username')
+          expect(page).to have_field('password')
+          expect(page).to have_button(I18n.t('auth.login'))
```

Comment:
actually, I don't think we need this test as the [fill form](https://github.com/gutakk/ruby-google-scraper/pull/5/files#diff-48177ea7c8836c9471826b4dae6a69b6R39) already cover this, the field should be displayed there

Reply:
Then do we still need `validates screen` context?

Reply:
Fixed, removed `have_field` tests.